### ISSUE TITLE
Remove divisions from scos-sensor and associated calibration file

### ISSUE THREAD

**Issue by ToddSchumannNTIA** [NTIA/scos-sensor#156]
To me, it does not make sense to have the divisions in scos-sensor or the associated calibration file. The only way to know where the divisions occur is to specifically look for them, in which case, they'd already have distinct points in the calibration file (i.e. the SDR would not be able to tune between divisions). If this was not done, then the user would not be able to include the divisions in the calibration file in the first place. 

Basically, the frequency divisions are already handled by judicious choice of calibration frequencies, so there's no need to include them in the calibration file separately.

**Comment by aromanielloNTIA** [comment]
SCOS Sensor no longer holds calibration files (they are instead in SCOS Actions or the signal analyzer plugin). I will open a corresponding issue to remove divisions from SCOS Actions, and close this issue.
